vnext/Desktop/WebSocketResourceFactory.cpp

```diff
@@ -3,14 +3,16 @@
 
 #include "pch.h"
 
+#include <Networking/WinRTWebSocketResource.h>
 #include <RuntimeOptions.h>
-#include <WinRTWebSocketResource.h>
+#if ENABLE_BEAST
 #include "BeastWebSocketResource.h"
+#endif // ENABLE_BEAST
 
 using std::shared_ptr;
 using std::string;
 
-namespace Microsoft::React {
+namespace Microsoft::React::Networking {
```

Comment:
>Networking [](http://example.com/codeflow?start=28&length=10)

@JunielKatarn it would be great if refactoring changes would be submitted separately from code logic changes. Otherwise, it's incredibly difficult for code reviewers to follow. Would it be difficult to split this PR into 2, one for refactoring, the other one for actual logic changes?

Reply:
It would be some extra work.

Moreover, this change is still going to modify the DLL boundary, so on the other hand it's a good opportunity to "batch" the changes together.